test/setup/main.tf

```diff
@@ -35,15 +35,22 @@ module "gke-project-1" {
     "cloudkms.googleapis.com",
     "cloudresourcemanager.googleapis.com",
     "container.googleapis.com",
+    "compute.googleapis.com",
```

Comment:
We removed this on purpose as it was increasing test flakiness. ref: https://github.com/hashicorp/terraform-provider-google/issues/7103

Reply:
I added it to try to fix the error
```
Error: googleapi: Error 403: Google Compute Engine: Required 'compute.regions.get' permission for 'projects/ci-gke-3ba695b1-b5d9/regions/us-central1'., forbidden
```
Adding the api didn't fix the problem. I will remove compute.googleapis.com